extensions/guacamole-auth-sso/modules/guacamole-auth-sso-saml/src/main/java/org/apache/guacamole/auth/saml/acs/SAMLService.java

```diff
@@ -98,6 +116,9 @@ public URI createRequest() throws GuacamoleException {
             throw new GuacamoleServerException("SAML IdP redirect could not "
                     + "be generated due to an error in the URI syntax: "
                     + e.getMessage());
+        } catch (SettingsException e) {
+            throw new GuacamoleServerException("SAML Settings errors: "
```

Comment:
Is there a more explanative contextual message we can provide here beyond "SAML Settings errors: ..."?

Reply:
Sure, I dug into the source to find the situation where it would throw this error. Will update.